src/Controller/DefaultController.php

```diff
@@ -55,9 +56,9 @@ public function index()
     /**
      * @Route("/admin", name="admin_home")
      */
-    public function adminIndex()
+    public function adminIndex(ContextTemplateService $contextTemplate)
     {
-        return $this->render('admin/index.html.twig');
+        return $this->render($contextTemplate->GetProgramTemplate('admin/index.html.twig'));
```

Comment:
Also, no need to change this as we can use the nph_admin_home route that I have created in the NphAdminController. You could just render the twig template that you have created for the nph admin.